.github/workflows/cpp.yml

```diff
@@ -67,7 +67,8 @@ jobs:
             image: conda-cpp
             llvm: "14"
             runs-on: ubuntu-latest
-            title: AMD64 Conda C++
+            simd-level: AVX2
+            title: AMD64 Conda C++ AVX2
```

Comment:
I don't object the suggestion but I like "title: AMD64 Conda C++ AVX2" approach because:

* Other jobs don't define `simd-level`
* It confuses a bit that `title` isn't used as-is

Reply:
@kou Good points, I'm going to make the change :-)